checker/raw_result.go

```diff
@@ -286,7 +287,8 @@ type BinaryArtifactData struct {
 // SignedReleasesData contains the raw results
 // for the Signed-Releases check.
 type SignedReleasesData struct {
-	Releases []clients.Release
+	Releases               []clients.Release
+	ProjectPackageVersions packageclient.ProjectPackageVersions
```

Comment:
most of the other checks define the subset of data they care about within this `checker` package. Right now `ProjectPackageVersions` re-uses the exact struct from the client, which seems very tied to the deps.dev API response.

Also, stylistically can we just call this `Packages` or something?